server/index.js

```diff
@@ -19,6 +19,26 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  const sql = `
+    select "productId",
+          "name",
+          "price",
+          "image",
+          "shortDescription"
+      from "products";
+  `;
+
+  db.query(sql)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(error => {
+      console.error(error);
```

Comment:
You can omit this console error log, because it happens for you in the error handling middleware defined just before `app.listen()`